src/Angular.js

```diff
@@ -893,7 +893,35 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @name ng.directive:ngJq
+ *
+ * @element html
+ * @param {string} (optional) the name of the library available under `window` to be used for angular.element
+ *
+ * @description
+ *
+ * Use this directive to force the angular.element library.  This should be used to force
+ * either jqLite by leaving ng-jq blank or setting the name of the jquery variable under window
+ * (eg. jQuery).
+ *
+ * This directive is global for the whole of the angular library, hence the directive can only
+ * be added to the top-most HTML element.
```

Comment:
This is not true as you're catching `ng-jq` anywhere in the document. I'd rather copy the text from `ngCsp.js`:

"put the `ngJq` directive on the root element of the application or on the `angular.js` script tag, whichever appears first in the html document."

so that it's advised to put it there but not claim it won't work otherwise.